tasks/transform_conformance/tests/esbuild-tests/test/fixtures/regexp/all-regex-plugins-enabled-by-targets/output.js

```diff
@@ -0,0 +1,9 @@
+x1 = new RegExp(".", "y");
+x2 = new RegExp(".", "u");
+a1 = new RegExp("a.b", "s");
+b1 = new RegExp("(?<!x)", "");
+b2 = new RegExp("(?<=x)", "");
+c1 = new RegExp("(?<a>b)", "");
+d1 = new RegExp("\\\\\\\\p{Emoji}", "u");
+f1 = new RegExp("\\\\\\\\p{Emoji}", "u");
+g1 = new RegExp("[\\\\\\\\p{White_Space}&&\\\\\\\\p{ASCII}]", "v");
```

Comment:
Are we having too many backslashes? esbuild generates `g1 = new RegExp("[\\\\p{White_Space}&&\\\\p{ASCII}]", "v");`